tests/jobserver/test_views.py

```diff
@@ -76,26 +88,70 @@ def test_joblist_filter_by_workspace(rf):
 
 @pytest.mark.django_db
 def test_joblist_search_by_action(rf):
-    job1 = JobFactory(action_id="run")
-    JobFactory(action_id="leap")
+    request1 = JobRequestFactory()
+    JobFactory(request=request1, action_id="run")
+
+    request2 = JobRequestFactory()
+    JobFactory(request=request2, action_id="leap")
 
     request = rf.get("/?q=run")
     response = JobList.as_view()(request)
 
     assert len(response.context_data["object_list"]) == 1
-    assert response.context_data["object_list"][0] == job1
+    assert response.context_data["object_list"][0] == request1
 
 
 @pytest.mark.django_db
 def test_joblist_search_by_id(rf):
-    JobFactory()
-    job2 = JobFactory(id=99)
+    JobFactory(request=JobRequestFactory())
+
+    request2 = JobRequestFactory()
+    JobFactory(request=request2, id=99)
 
     request = rf.get("/?q=99")
     response = JobList.as_view()(request)
 
     assert len(response.context_data["object_list"]) == 1
-    assert response.context_data["object_list"][0] == job2
+    assert response.context_data["object_list"][0] == request2
+
+
+@pytest.mark.django_db
+def test_jobrequestcreate_redirects_to_new_job(client):
+    client.force_login(UserFactory())
+    response = client.post("/jobs/new/0/", follow=True)
+
+    assert response.status_code == 200
+    assert response.redirect_chain == [(reverse("job-select-workspace"), 302)]
+
+    messages = list(response.context["messages"])
+    assert len(messages) == 1
+    assert str(messages[0]) == "Unknown Workspace, please pick a valid one"
+
+
+@pytest.mark.django_db
+def test_jobrequestcreate_success(rf):
+    user = UserFactory()
+    workspace = WorkspaceFactory()
+
+    data = {
+        "action_id": "twiddle",
+        "backend": "test",
+        "callback_url": "test",
+    }
+
+    request = rf.post("/", data)
```

Comment:
Can we do something here to indicate that the slash is meaningless? e.g. a `MEANINGLESS_URL` constant or better name?